packages/documentation/src/styles/rafiki.css

```diff
@@ -54,12 +65,60 @@
   text-align: center;
 }
 
-.sl-markdown-content :is(h1, h2, h3, h4, h5, h6) > a {
-  color: inherit;
-  text-decoration: none;
-  transition: color 0.2s ease-in-out;
+/* Overflowing table styles */
+.overflow-table {
+  overflow-x: scroll;
+  background:
+    linear-gradient(to right, white 30%, rgba(255, 255, 255, 0)),
+    linear-gradient(to right, rgba(255, 255, 255, 0), white 70%) 0 100%,
+    radial-gradient(
+      farthest-side at 0% 50%,
+      rgba(0, 0, 0, 0.2),
+      rgba(0, 0, 0, 0)
+    ),
+    radial-gradient(
+        farthest-side at 100% 50%,
+        rgba(0, 0, 0, 0.2),
+        rgba(0, 0, 0, 0)
+      )
+      0 100%;
+  background-repeat: no-repeat;
+  background-color: white;
+  background-size:
+    40px 100%,
+    40px 100%,
+    14px 100%,
+    14px 100%;
+  background-position:
+    0 0,
+    100%,
+    0 0,
+    100%;
+  background-attachment: local, local, scroll, scroll;
 }
 
-.sl-markdown-content :is(h1, h2, h3, h4, h5, h6) > a:hover {
-  color: var(--sl-color-accent-high);
+@media screen and (min-width: 80rem) {
+  .overflow-table {
+    max-width: 755px;
+  }
+}
+
+.overflow-table table code {
+  white-space: nowrap;
+}
+
+.overflow-table table tbody tr:nth-child(2n):not(:where(.not-content *)) {
+  background-color: initial;
+}
+
+.overflow-table tbody tr:not(:last-of-type) td {
+  border-block-end: 1px solid var(--sl-color-gray-6);
```

Comment:
Since the background colour was removed, a bottom border helps make the rows look distinct from each other.